modulemd_tools/modulemd_tools/yaml.py

```diff
@@ -0,0 +1,183 @@
+"""
+Module for working with modulemd YAML definitions with the least abstractions as
+possible. Within this module, modulemd YAMLs are represented simply as a string,
+and all transformation functions are `str` -> `str`.
+"""
+
+import os
+import gi
+gi.require_version("Modulemd", "2.0")
+from gi.repository import Modulemd
+
+
+def is_valid(yaml):
+    """
+    Determine whether the `yaml` string is a valid modulemd YAML definition
+    """
+    idx = Modulemd.ModuleIndex.new()
+    (ret, _) = idx.update_from_string(yaml, strict=True)
```

Comment:
The parentheses are not necessary, they even decrease code readability.
I'd use ```ret, _ = ...```